src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -277,12 +277,15 @@ private Bucket newCloudBucket(GoogleBucketResource bucketResource) {
     boolean doVersioning =
         Arrays.stream(env.getActiveProfiles()).noneMatch(env -> env.contains("test"));
     String bucketName = bucketResource.getName();
+    GoogleRegion region = bucketResource.getRegion();
+    StorageClass storageClass =
+        region.isMultiRegional() ? StorageClass.MULTI_REGIONAL : StorageClass.REGIONAL;
```

Comment:
These are legacy storage classes and according to Google we should use `STANDARD` instead, but that can definitely go into a future tech debt ticket: https://cloud.google.com/storage/docs/storage-classes